src/utils/__tests__/makeReactNativeConfig.test.js

```diff
@@ -33,3 +33,70 @@ test('merges existing config', () => {
     'merges existing config'
   );
 });
+
+describe('injects polyfill into entries, when', () => {
```

Comment:
You sure this is named properly? Should be `describe("name of feature")` and then, `it("should inject")`, no? @thymikee

Reply:
should be whatever suites best ;)